Xamarin.Forms.Platform.Android/Material/MaterialActivityIndicatorRenderer.cs

```diff
@@ -179,7 +178,7 @@ SizeRequest IVisualElementRenderer.GetDesiredSize(int widthConstraint, int heigh
 		}
 
 		void IVisualElementRenderer.SetElement(VisualElement element) =>
-			Element = (element as ActivityIndicator) ?? throw new ArgumentException("Element must be of type ActivityIndicator.");
+			Element = (element as ActivityIndicator) ?? throw new ArgumentException($"{nameof(element)} must be of type {typeof(ActivityIndicator).Name}.");
```

Comment:
Why is this `typeof().Name` instead of just `nameof()`?

Reply:
Hm. I think the message should include the name of the type that element is because it's that type the user assigned which needs to be an ActivityIndicator.